CMakeLists.txt

```diff
@@ -161,23 +168,16 @@ function(spvtools_opencl_tables VERSION)
   add_custom_target(spirv-tools-build-opencl-tables-${VERSION}
     ${PYTHON_EXECUTABLE}
     ${CMAKE_CURRENT_SOURCE_DIR}/utils/generate_grammar_tables.py
-    --spirv-core-grammar=${spirv-tools_SOURCE_DIR}/source/spirv-${VERSION}.core.grammar.json
+    --spirv-core-grammar=${SPIRV_HEADER_INCLUDE_DIR}/spirv/${VERSION}/spirv.core.grammar.json
     --extinst-opencl-grammar=${spirv-tools_SOURCE_DIR}/source/extinst-${VERSION}.opencl.std.grammar.json
     --opencl-insts-output=${spirv-tools_BINARY_DIR}/opencl.std.insts-${VERSION}.inc
     COMMENT "Generate info tables for OpenCL extended instructions and operands v${VERSION}.")
 endfunction(spvtools_opencl_tables)
 
-spvtools_core_tables("1-0")
-spvtools_core_tables("1-1")
-spvtools_opencl_tables("1-0")
-spvtools_glsl_tables("1-0")
-
-# Defaults to OFF if the user didn't set it.
-option(SPIRV_SKIP_EXECUTABLES
-       "Skip building the executable and tests along with the library"
-       ${SPIRV_SKIP_EXECUTABLES})
-
-add_subdirectory(external)
+spvtools_core_tables("1.0")
```

Comment:
Replacing `-` with `.` as a separator is OK, I guess, but it made this change larger than it needs to be.  Is there a significant benefit that I'm missing?